hs/t/y/object_set6.rsh

```diff
@@ -0,0 +1,17 @@
+'reach 0.1';
+
+export const main = Reach.App(() => {
```

Comment:
You should do an example that changes the type

Reply:
I'm not sure what you mean. Don't 4/6 of these examples change the type?

Reply:
I'm just dumb then and couldn't tell